app/controllers/forgot_passwords_controller.rb

```diff
@@ -1,23 +1,18 @@
 class ForgotPasswordsController < ApplicationController
 
-  def new
-
-  end
+  def new; end
 
   def create
-    user = User.where(email: params[:email]).take
+    user = User.find_by(email: params[:email])
     if user
       AppMailer.send_forgot_password_email(user).deliver
       redirect_to forgot_password_confirm_path
-      return
     else
       flash[:error] = params[:email].blank? ? 'email input can not be blank.' : 'email is not existed.'
```

Comment:
Be careful about telling too much in your error messages. A hacker now knows that this email does not exist. That means they will also know when they guess one that does exist.